base/src/main/java/io/spine/validate/FieldValue.java

```diff
@@ -169,14 +169,14 @@ JavaType javaType() {
      * @param <O>
      *         the type of the option value
      */
-    <O> Option<O> option(GeneratedExtension<FieldOptions, O> option) {
-        Optional<Option<O>> validationRuleOption = getOptionValue(context, option);
+    <O> O option(GeneratedExtension<FieldOptions, O> option) {
```

Comment:
Please have another letter for the generic parameter. `O` looks like zero. Candidates are: `V` (for value), `F` (for field).